result.go

```diff
@@ -296,18 +303,16 @@ func newResultObjectField(idx int, f reflect.StructField) (resultObjectField, er
 
 	default:
 		var err error
-		r, err = newResult(f.Type)
+		if name := f.Tag.Get(_nameTag); len(name) > 0 {
+			// can modify in-place because options are passed-by-value.
+			opts.Name = name
+		}
+		r, err = newResult(f.Type, opts)
 		if err != nil {
 			return rof, err
 		}
 	}
 
-	if rs, ok := r.(resultSingle); ok {
```

Comment:
groups don't have names right?

Reply:
groups and names are mutually exclusive.